packages/renderer/src/lib/kube/KubernetesDashboardGuideCard.svelte

```diff
@@ -10,6 +10,7 @@ interface Props {
 let { title, link, image }: Props = $props();
 
 async function openLink(): Promise<void> {
+  await window.telemetryTrack('kubernetes.dashboard.guide', { title: title, link: link });
   await window.openExternal(link);
```

Comment:
```suggestion
  try {
    await window.openExternal(link);
  } finally {
    await window.telemetryTrack('kubernetes.dashboard.guide', { title: title, link: link });
  }
```

Telemetry is secondary, so we should probably have it sent in a finally block, to avoid any delay, errors etc.

Reply:
Agreed, fixed.